components/StyledListContainer/index.js

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+
+const StyledListContainer = styled.ul`
+  list-style: none;
+  margin: 0;
```

Comment:
It appears that the margin property has no effect on your list at the moment. It's easy to accidentally include unnecessary CSS properties when styling.
**There's a nice little tipp:** To check CSS properties, you can use the developer tools to test each property individually. 

```suggestion

```